Dockerfile.bindings

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu@sha256:19478ce7fc2ffbce89df29fea5725a8d12e57de52eb9ea570890dc5852aac1ac
```

Comment:
Maybe move this file to `dev/`? That's where all the other development scripts live. Unless that makes it harder to do some other things for some reason.